src/main/kotlin/com/atlassian/performance/tools/report/api/result/RawCohortResult.kt

```diff
@@ -26,10 +28,24 @@ abstract class RawCohortResult private constructor() {
     /**
      * Prepares post-processed performance results.
      */
+    @Deprecated(
+        message = "Use prepareForJudgement with two arguments instead.",
+        replaceWith = ReplaceWith(
+            expression = "prepareForJudgement(timeline, actionTypeToOutlierTrimming)"
```

Comment:
Current callers don't have `actionTypeToOutlierTrimming`. Specifying it is also extra work. Some didn't need it before and won't need it now, but the cost is applied to all.
Perhaps we should leave this method be and not deprecate it.